functions/marks.fish

```diff
@@ -15,15 +15,12 @@ function marks
       if test (count $mark_list) -eq 0
         echo "No marks currently defined."
       else
-        set -l current_dir (pwd)
         set -l output ""
         for mark_name in $mark_list
-          cd $MARKPATH/$mark_name
-          set -l real_path (pwd)
+          set -l real_path (readlink $MARKPATH/$mark_name)
```

Comment:
I'm not sure how portable `readlink` is. fish has a built-in `realpath` function to accomplish the same thing: https://fishshell.com/docs/current/commands.html#realpath

Reply:
Thanks for the suggestion. I changed the function to use realpath.